csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/OnnxMl.cs

```diff
@@ -5819,6 +6144,7 @@ public sealed partial class Tensor : pb::IMessage<Tensor>
         /// This field MUST be present for this version of the IR.
         /// </summary>
         [global::System.Diagnostics.DebuggerNonUserCodeAttribute]
+        [global::System.CodeDom.Compiler.GeneratedCode("protoc", null)]
```

Comment:
Is this related to your changes?

Reply:
I accidentally merged this with the ONNX 1.12 change. Once I merge ONNX 1.12 we will rebase this on master and these changes should go away.